sdk/ai/azure-ai-inference/samples/sample_chat_completions_from_input_prompty.py

```diff
@@ -0,0 +1,69 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""
+DESCRIPTION:
+    This sample demonstrates how to get a chat completions response from
+    the service using a synchronous client, and directly providing the 
+    input in Prompty format.
+
+    This sample assumes the AI model is hosted on a Serverless API or
+    Managed Compute endpoint. For GitHub Models or Azure OpenAI endpoints,
+    the client constructor needs to be modified. See package documentation:
+    https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ai/azure-ai-inference/README.md#key-concepts
+
+USAGE:
+    python sample_chat_completions_from_input_prompty.py
+
+    Set these two environment variables before running the sample:
+    1) AZURE_AI_CHAT_ENDPOINT - Your endpoint URL, in the form 
+        https://<your-deployment-name>.<your-azure-region>.models.ai.azure.com
+        where `your-deployment-name` is your unique AI Model deployment name, and
+        `your-azure-region` is the Azure region where your model is deployed.
+    2) AZURE_AI_CHAT_KEY - Your model key (a 32-character string). Keep it secret.
+"""
+# mypy: disable-error-code="union-attr"
+# pyright: reportAttributeAccessIssue=false
+
+
+def sample_chat_completions_from_input_prompty():
+    import os
+    from azure.ai.inference import ChatCompletionsClient
+    from azure.ai.inference.prompts import PromptyTemplate
+    from azure.core.credentials import AzureKeyCredential
+
+    try:
+        endpoint = os.environ["AZURE_AI_CHAT_ENDPOINT"]
+        key = os.environ["AZURE_AI_CHAT_KEY"]
+    except KeyError:
+        print("Missing environment variable 'AZURE_AI_CHAT_ENDPOINT' or 'AZURE_AI_CHAT_KEY'")
+        print("Set them before running this sample.")
+        exit()
+
+
+    path = "./sample1.prompty"
+    prompt_config = PromptyTemplate.load(file_path=path)
+
+    input_variables = {
+        "input": "please tell me a joke about cats",
+    }
+
+    messages = prompt_config.render(input_variables=input_variables)
+    # messages = prompt_config.render(input_variables=input_variables, format="openai")
```

Comment:
What is the 2nd line, with format="openai"? Looks like it need to be removed.

Reply:
Can you suggest what's the best way to demonstrate the sample parameters? -- `format` can be either empty, or "openai", if it's empty, the response is the array of Inference SDK message array (array of objects, e.g. SystemMessage); if we specify it's "openai", the output is array of Python objects -- also maybe "openai" is not a good name as a lot of other models may take the same input as well

Reply:
"Format" parameter is removed